docs/datadog_monitor.md

```diff
@@ -0,0 +1,134 @@
+# Getting Started
+
+This procedure describes the simplest and fastest way to deploy a `DatadogMonitor` with the Datadog Operator. **Note: Operator version 0.6+ is required.**
+
+## Prerequisites
+
+Using the Datadog Operator requires the following prerequisites:
+
+- **Kubernetes Cluster version >= v1.14.X**: Tests were done on versions >= `1.14.0`. However, it should work on versions `>= v1.11.0`. For earlier versions, due to limited CRD support, the Operator may not work as expected.
+- [`Helm`][1] for deploying the `datadog-operator`.
+- [`Kubectl` cli][2] for installing a `DatadogMonitor`.
+
+## Adding a DatadogMonitor
+
+To deploy a `DatadogMonitor` with the Datadog Operator, see the [`datadog-operator`][3] Helm chart.
+Here are the steps:
+
+1. Install the [Datadog Operator][4]:
```

Comment:
Isn't that already covered in the Prerequisites? Feels like dup, maybe keep this and remove the above